src/templates/ObservabilityPackDetails.js

```diff
@@ -167,48 +148,46 @@ const ObservabilityPackDetails = ({ data, location }) => {
                     )}
                   </>
                 ))}
+                <Intro
+                  css={css`
+                    margin-bottom: 16px;
+                  `}
+                >
+                  {pack.name} observability pack contains{' '}
+                  {pluralize('dashboard', pack.dashboards?.length ?? 0, true)}.
+                  These interactive visualizations let you easily explore your
+                  data, understand context, and resolve problems faster.
+                </Intro>
               </Tabs.Page>
               <Tabs.Page id="alerts">
                 <Intro
                   css={css`
                     margin-bottom: 16px;
                   `}
                 >
-                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-                  Ut enim ad minim veniam, quis nostrud exercitation ullamco
-                  laboris nisi ut aliquip ex ea commodo consequat. Duis aute
-                  irure dolor in reprehenderit in voluptate velit esse cillum
-                  dolore eu fugiat nulla pariatur. Excepteur sint occaecat
-                  cupidatat non proident, sunt in culpa qui officia deserunt
-                  mollit anim id est laborum.
+                  {pack.name} observability pack contains{' '}
+                  {pluralize('alert', pack.alerts?.length ?? 0, true)}, which
+                  detect changes in key performance metrics. Integrate these
```

Comment:
I'm back with plural issues 🙈, if alert is singular I believe this should say "detects" whereas plural alerts could be "detect". maybe we just want to reword this rather than pluralize detect. the rest say "these ______" so maybe we say "these alerts detect" and call it good?

Reply:
sounds good 👍

Reply:
updated